styles/catalogo.css

```diff
@@ -0,0 +1,3 @@
+.catalogue-main{
+    margin-inline: 2em;
+}
```

Comment:
I don't think this classname is really necessary. The `mx` style prop from Chakra should be enough: https://chakra-ui.com/docs/features/style-props
<img width="857" alt="Screenshot 2021-12-29 at 01 06 47" src="https://user-images.githubusercontent.com/26777462/147616045-572f231a-2027-4670-8e02-7e90ff11a174.png">